sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -31,6 +31,7 @@
 import com.google.common.collect.Iterators;
 import com.google.common.collect.Lists;
 import com.google.common.io.Closeables;
+
```

Comment:
kind of code standard, same for the added `@Override` in KafkaIO, intend to keep it for better reading.